src/sawyer_control/environments/sawyer_pushing.py

```diff
@@ -109,87 +56,23 @@ def compute_her_reward_np(self, observation, action, next_observation, goal):
         :param goal:
         :return:
         '''
-        return -np.linalg.norm(next_observation[21:24] - goal)
+        return 0
 
-    def reset(self, vae_reset=False):
-        self.in_reset = True
-        # self._safe_move_to_neutral()
+    def reset(self):
         self.thresh = False
-        self._joint_act(self.reset_position  - self._end_effector_pose()[:3])
-        self.in_reset = False
+        self._act(self.reset_position  - self._end_effector_pose()[:3])
         self.thresh = True
         if self.pause_on_reset:
             #PAUSE AND MOVE OBJECT TO ITS RESET POSITION
             print('in reset')
             input()
-        if self.img_observation:
-            observation = self.get_image()
-        else:
-            observation = self._get_observation()
+        observation = self._get_observation()
         return observation
 
-    def _joint_act(self, action):
-        ee_pos = self._end_effector_pose()
-        if self.relative_pos_control:
-            target_ee_pos = (ee_pos[:3] + action)
-        else:
-            target_ee_pos = action
-        target_ee_pos[2] = self.z
-        target_ee_pos[:2] = np.clip(target_ee_pos[:2], self.pushing_box_low, self.pushing_box_high)
-        target_ee_pos = np.concatenate((target_ee_pos, ee_pos[3:]))
-        angles = self.request_ik_angles(target_ee_pos, self._joint_angles())
-        self.send_angle_action(angles)
-
-    def _get_observation(self):
-        angles = self._joint_angles()
-        _, velocities, torques, _ = self.request_observation()
-        velocities = np.array(velocities)
-        torques = np.array(torques)
-        endpoint_pose = self._end_effector_pose()
-
-        temp = np.hstack((
-            angles,
-            velocities,
-            torques,
-            endpoint_pose,
-        ))
-        return temp
-
-    def step(self, action):
-        self._act(action)
-        observation = self._get_observation()
-        reward = self.reward() * self.reward_magnitude
-        done = False
-        info = {}
-        #info['cartesian_goal'] = self.desired.copy()
-        #pos = np.array(observation[21:24])
-        #info['distance_to_goal'] = np.linalg.norm(pos - self.desired)
-        if self.img_observation:
-            observation = self.get_image()
-        return observation, reward, done, info
+    def _get_info(self):

```

Comment:
this should return logging data, but if you don't want it to return anything then just delete this function, by default the base returns an empty dict